dipy/align/metrics.py

```diff
@@ -244,6 +244,21 @@ def initialize_iteration(self):
         re-orienting the gradients in the voxel space using the corresponding
         affine transformations.
         """
+        min_size = self.radius * 2 + 1
+
+        def invalid_image_size(image):
+            valid_shape = [size for size in image.shape if
+                           size >= min_size]
+            return True if len(valid_shape) != image.ndim else False
+
+        msg = ("Each image dimensions should be superior to 2 * radius + 1."
+               "Decrease CCMetric radius or Increase your image size")
+
+        if invalid_image_size(self.static_image):
+            raise IOError("Static image size is too small. " + msg)
+        if invalid_image_size(self.moving_image):
+            raise IOError("Moving image size is too small. " + msg)
```

Comment:
I would have expected `ValueError` or `RuntimeError` rather than `IOError`, but I'm not familiar with what convention DiPy tends to favor.